cadc-rest/src/main/java/ca/nrc/cadc/rest/RestServlet.java

```diff
@@ -115,6 +115,8 @@ public void init(ServletConfig config) throws ServletException
         this.putAction = loadAction(config, "put");
         this.deleteAction = loadAction(config, "delete");
         this.headAction = loadAction(config, "head");
+        this.appName = config.getServletContext().getServletContextName();
+        this.restEndpoint = config.getServletContext().getServletContextName()  + "." + config.getServletName();
```

Comment:
Shouldn't this be separated by a slash instead of a period?

Reply:
I really didn't want them to look like paths since they are not - these names are independent of servlet-mappings - they are just named components.

Reply:
renaming restEndpoint to componentID